datasets/irs990.yaml

```diff
@@ -51,4 +51,8 @@ DataAtWork:
       URL: https://www.grantmakers.io
       AuthorName: Chad Kruse
       AuthorURL: https://www.chadkruse.com/
+    - Title: npo_classifier
```

Comment:
Thanks @ma-ji! Could you just add a little bit more of a description to the `Title` field and then this should be good to merge in?

Reply:
Sure thing, just did that!